### DISCUSSION TITLE
Migrate emails from one server to another

### DISCUSSION THREAD

**Discussion by Rol-a** [modoboa/modoboa#2395] [Q&A]
@tonioo
Hello Sir. I have a situation here and hopefully I can get a hint from you or anyone else in order to move in the right direction.
I initially had a lets-encrypt issue with my instance, but never was able to fully fix it. So I decide to fire up a fresh server and reinstall everything. Is it then possible to migrate the current emails (sent and received) to the new instance? Notice that the IP will change, but the domain will remain the same.
Thanks in advance.

**Comment by tonioo** [comment]
You can use a tool such a rsync to move emails from the old server to the new one.
